ChopChop/Models/Ingredients/Ingredient/OnlineRecipeIngredient.swift

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
tbh i think can just use RecipeIngredient
ANYWAYS the nested models like RecipeIngredient and RecipeStep are not supposed to have ids anyway